db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180813141443) do
+ActiveRecord::Schema.define(version: 20180831000000) do
```

Comment:
This schema version doesn't seem to be related to any migration files. I wonder how it got there :confused:.